sections/exchange/TradeCard/Charts/CombinedPriceChartCard.tsx

```diff
@@ -125,115 +164,90 @@ const ChartCard: FC<ChartCardProps> = ({
 							)}
 							{change != null && <ChangePercent value={change} />}
 						</>
-					) : (
-						<CurrencyLabel>{t('common.price')}</CurrencyLabel>
 					)}
-				</FlexDivRowCentered>
+				</ChartHeaderInner>
 				{!isMarketClosed && (
 					<Actions>
-						{PERIOD_LABELS.map((period) => (
-							<StyledTextButton
-								key={period.value}
-								isActive={period.value === selectedPeriod.value}
-								onClick={() => setSelectedPeriod(period)}
-							>
-								{t(period.i18nLabel)}
-							</StyledTextButton>
-						))}
+						<CompareRatioToggleContainer>
+							<CompareRatioToggle>
+								{eitherCurrencyIsSUSD ? null : (
+									<CompareRatioToggleType
+										onClick={() => {
+											setSelectedChartType(ChartType.COMPARE);
+										}}
+										isActive={isCompareChart}
+									>
+										{t('common.chart-types.compare')}
+									</CompareRatioToggleType>
+								)}
+								<CompareRatioToggleType
+									onClick={() => {
+										setSelectedChartType(ChartType.AREA);
+									}}
+									isActive={isAreaChart}
+								>
+									{t('common.chart-types.ratio')}
+								</CompareRatioToggleType>
+								{selectedPeriod !== Period.ONE_MONTH ? null : (
+									<CompareRatioToggleType
```

Comment:
I think you should refactor `ChartTypeToggle` to use this text version of the toggle and use it on both the Double and Single view. That way we are more consistent.

Reply:
Fixed.